exercises/protein-translation/src/test/scala/ProteinTranslationTest.scala

```diff
@@ -0,0 +1,65 @@
+import org.scalatest.{FunSuite, Matchers}
```

Comment:
Have a PR to add `canonical-data.json` first?

Although this will probably trigger a lengthy discussion. :)